core/src/main/java/com/datastrato/gravitino/authorization/AuthorizationUtils.java

```diff
@@ -69,4 +69,40 @@ public static Namespace ofGroupNamespace(String metalake) {
   public static Namespace ofUserNamespace(String metalake) {
     return Namespace.of(metalake, Entity.SYSTEM_CATALOG_RESERVED_NAME, Entity.USER_SCHEMA_NAME);
   }
+
+  public static void checkUser(NameIdentifier ident) {
+    NameIdentifier.check(ident != null, "User identifier must not be null");
+    checkUserNamespace(ident.namespace());
+  }
+
+  public static void checkGroup(NameIdentifier ident) {
+    NameIdentifier.check(ident != null, "Group identifier must not be null");
+    checkGroupNamespace(ident.namespace());
+  }
+
+  public static void checkRole(NameIdentifier ident) {
+    NameIdentifier.check(ident != null, "Role identifier must not be null");
```

Comment:
`must not` -> `can't`

Reply:
i copy the error msg from `NameIdentifier`, maybe we should be aligned with other msg ?

Reply:
Ok for this.